jQueryI.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+		<!--<script type="text/javascript" src="jQueryI.js"></script>
+		<script type="text/javascript" src="FileStructure.js"></script>-->
+		<link rel="stylesheet" href="FileStructure.css">
+	</head>
+	<body>
+		<p>Political Allegiance:<br>
+  			<input type="text" id="politicalAllegiance"><br>
+  			Citizenship<br>
+  			<input type="text" id="citizenship"><br>
+  			<button id="Submit">Submit</button>
+		</p>
+		<script type="text/javascript" src="jQueryI.js"></script>
```

Comment:
See the notes about file names at 

https://github.com/nanohackers/js-02-class-f17/blob/master/Class%20Notes/challenge2Notes.md#suggested-file-structure

There are suggestions about what to name your files, and how to organize them. Since you seem to be making separate files for each challenge (great idea - you'll be able to go back to them at any time!), you might consider putting each challenge into a separate folder. Something like this:

```
/challenge2
   index.html
   /scripts
      script.js
   /styles
      style.css
/challenge3
   index.html
   /scripts
      script.js
   /styles
      style.css
... etc
```